cmd/lakectl/cmd/common_helpers.go

```diff
@@ -293,6 +293,19 @@ func MustParseRefURI(name, s string) *uri.URI {
 	return u
 }
 
+func MustParseRefWithPathURI(name, s string) *uri.URI {
```

Comment:
Not the same as `MustParsePathURI`?

Reply:
MustParsePathURI requires path element to not be empty

Reply:
Having the same conversation, I'll use this line:
why the path is optional? if we request a path from the use like in other commands the use of '/' after the ref should work as a valid path and the use of MustparsePathURI will work.

Reply:
I had this discussion with @guy-har in a different PR. Initially we used MustParsePathURI but examples in lakectl were given where we don't enforce providing the path.
I tend to go either way - maybe @guy-har can also comment

Reply:
On second thought - I tend to agree with this approach. Will use MustParsePathURI - users will need to pass a `\` delimiter when providing the ref root